src/runtime/Makefile

```diff
@@ -112,12 +114,15 @@ PKGLIBEXECDIR := $(LIBEXECDIR)/$(PROJECT_DIR)
 KERNELDIR := $(PKGDATADIR)
 
 IMAGEPATH := $(PKGDATADIR)/$(IMAGENAME)
+INITRDPATH := $(PKGDATADIR)/$(INITRDNAME)
 FIRMWAREPATH :=
 FIRMWAREVOLUMEPATH :=
 TDVFFIRMWAREPATH := $(PREFIXDEPS)/share/tdvf/OVMF_CODE.fd
 TDVFFIRMWAREVOLUMEPATH := $(PREFIXDEPS)/share/tdvf/OVMF_VARS.fd
 TDSHIMFIRMWAREPATH := ${PREFIXDEPS}/share/td-shim/td-shim.bin
+SEVFIRMWAREPATH := $(PREFIXDEPS)/share/sev/OVMF.fd
 TDXKERNELPARAMS := tdx_disable_filter
+SEVKERNELPARAMS := agent.config_file=/etc/default/agent.cfg
```

Comment:
We should try to standardize this path across the project.

Reply:
In regards to the `Makefile`, how does this look:

https://github.com/kata-containers/kata-containers/blob/72e46030851d2ca049d6b8629499c7bde65f2d4f/src/runtime/Makefile#L125-L129

Or do you want this set somewhere before the `Makefile`?

Added these changes for now.

Reply:
Updated agent config file to match how it is set in other locations in the codebase:

https://github.com/ryansavino/kata-containers/blob/9d4ffb92c2ff15f6377b150f2fb17ffa8e1f2641/src/runtime/Makefile#L125